### ISSUE TITLE
CHUNKLOADING

### ISSUE THREAD

**Issue by Elmani335** [FTBTeam/FTB-Utilities#1080]
Hello, so ranks.json have been updated to ranks.txt but now how can we change the max claimed and loaded chunks limit ? 
because i tryed everything and it's just not working maybe a file in the /config/ would be better

**Comment by Ashmata** [comment]
YES how can we limit chunks loaded by players or even disable it? Having the same issue! PLEASE HELP. we have been stuck on this for 2 weeks digging through cfg's and forums

**Comment by BrandonSoul** [comment]
Same here cant figure out how to change the amount of claimed chunks for teams

**Comment by Tahakk** [comment]
If you cannot change your ranks, post your rank.txt here so others can take a look
